notebooks/212-onnx-style-transfer/212-onnx-style-transfer.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c2cd5c88",
+   "metadata": {
+    "id": "moved-collapse"
+   },
+   "source": [
+    "# Style Transfer on ONNX Models with OpenVINO\n",
+    "\n",
+    "![Neural Style Transfer network output](https://user-images.githubusercontent.com/77325899/147354137-4fc9e79e-0195-4927-9608-0e3f17973d75.png)\n",
+    "\n",
+    "This notebook demonstrates [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) on ONNX models with OpenVINO. Neural style transfer networks take two images as input during training: a source image, and a style reference image. The network transforms the source image and returns an image with the style of the reference image applied to the source image.\n",
```

Comment:
```suggestion
    "This notebook demonstrates [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) on ONNX models with OpenVINO. Neural style transfer networks take two images as input during inference: a source image, and a style reference image. The network transforms the source image and returns an image with the style of the reference image applied to the source image.\n",
```
Should this say inference (not training)?

Reply:
No, during inference there is only one input image (that is why we have 5 different models for 5 different styles). The intention was to add a bit of explanation on how these models work, but if it adds confusion it is probably better to remove that sentence.

Reply:
Let's remove it then, since we don't show anything about the training in the notebook.

Reply:
Done.